platforms/common/include/px4_platform_common/shutdown.h

```diff
@@ -70,6 +70,12 @@ __EXPORT int px4_register_shutdown_hook(shutdown_hook_t hook);
  */
 __EXPORT int px4_unregister_shutdown_hook(shutdown_hook_t hook);
 
+/** Types of function calls supported by the worker task */
+typedef enum {
+	REBOOT_REQUEST = 0,          ///< Normal reboot
+	REBOOT_TO_BOOTLOADER,        ///< Reboot to PX4 bootloader
```

Comment:
These are shared. So to be To be dogmatic add = 2, and = 3

Reply:
2 PRs one for the rom changes 1 commit. The Second for the boot changes 2 commits. On commit changes the system adding the ISP and on adds the usages of the ISP on the imx